src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py

```diff
@@ -4016,7 +4016,18 @@ def test_bicep_list_versions(self):
 
 # Because don't want to record bicep cli binary
 class BicepBuildTest(LiveScenarioTest):
-    
+
+    def __init__(self, *arg, **kwargs):
+        super().__init__(*arg, random_config_dir=True, **kwargs)
```

Comment:
You also need to change `_bicep_installation_dir` to get `config_dir` from `self.cli_ctx.config.config_dir`.
Otherwise it still installs bicep to same directory.
```
_config_dir = get_config_dir()
_bicep_installation_dir = os.path.join(_config_dir, "bin")
```

Reply:
Thanks! Since the tests are now serial, I suppose the `random_config_dir` doesn't really add value now. I will create a separate PR for those changes then. 👍